sdk/inc/azure/core/az_http_transport.h

```diff
@@ -56,7 +56,16 @@ AZ_INLINE az_http_method az_http_method_delete() { return AZ_SPAN_FROM_STR("DELE
 AZ_INLINE az_http_method az_http_method_patch() { return AZ_SPAN_FROM_STR("PATCH"); }
 
 /**
- * @brief A type representing a buffer of #az_pair instances for HTTP request headers.
+ * @brief Represents a name/value pair of #az_span instances.
+ */
+typedef struct
+{
+  az_span name; ///< Name.
+  az_span value; ///< Value.
+} _az_http_request_header;
+
+/**
+ * @brief A type representing a buffer of #_az_http_header instances for HTTP request headers.
```

Comment:
What type is `#_az_http_header`? Should this be:
```suggestion
 * @brief A type representing a buffer of #_az_http_request_header instances for HTTP request headers.
```